### PR TITLE
[Layout API] Rename ASStaticLayoutSpec to ASAbsoluteLayoutSpec

### PR DESCRIPTION
**Pull request by hannahmbanana**
Renaming for clarity:

- ASStaticLayoutSpec —> ASAbsoluteLayoutSpec
- ASStaticLayoutable —> ASAbsoluteLayoutElement

"absolute" was chosen to replace "static" because the children of that layout spec are placed absolutely in relation to their parent.

### CODE REVIEW

**Comment by hannahmbanana** [comment]
@maicki: ready for review on Monday

File: AsyncDisplayKit.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1940,12 +1940,15 @@
 			isa = PBXNativeTarget;
 			buildConfigurationList = 058D09D2195D04C000B7D73C /* Build configuration list for PBXNativeTarget "AsyncDisplayKitTests" */;
 			buildPhases = (
+				D03848B30EFBD77CA805C258 /* 📦 Check Pods Manifest.lock */,
```
Comment (maicki) [inline AsyncDisplayKit.xcodeproj/project.pbxproj]: We should remove this kind of lines.

File: AsyncDisplayKit.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -1940,12 +1940,15 @@
...
 				2E61B6A0DB0F436A9DDBE86F /* [CP] Check Pods Manifest.lock */,
 				058D09B8195D04C000B7D73C /* Sources */,
 				058D09B9195D04C000B7D73C /* Frameworks */,
 				058D09BA195D04C000B7D73C /* Resources */,
 				3B9D88CDF51B429C8409E4B6 /* [CP] Copy Pods Resources */,
 				B130AB1AC0A1E5162E211C19 /* [CP] Embed Pods Frameworks */,
+				8F04C65981B095A7B88BC774 /* 📦 Embed Pods Frameworks */,
+				C0FD47294835C5E0D82246C8 /* 📦 Copy Pods Resources */,
```
Comment (maicki) [inline AsyncDisplayKit.xcodeproj/project.pbxproj]: Remove

File: AsyncDisplayKit.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -2076,6 +2079,21 @@
...
+			inputPaths = (
+			);
+			name = "📦 Embed Pods Frameworks";
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = "\"${SRCROOT}/Pods/Target Support Files/Pods-AsyncDisplayKitTests/Pods-AsyncDisplayKitTests-frameworks.sh\"\n";
+			showEnvVarsInLog = 0;
+		};
```
Comment (maicki) [inline AsyncDisplayKit.xcodeproj/project.pbxproj]: Remove

File: AsyncDisplayKit.xcodeproj/project.pbxproj [hunk 4]
```diff
@@ -2091,6 +2109,36 @@
...
+			inputPaths = (
+			);
+			name = "📦 Check Pods Manifest.lock";
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = "diff \"${PODS_ROOT}/../Podfile.lock\" \"${PODS_ROOT}/Manifest.lock\" > /dev/null\nif [[ $? != 0 ]] ; then\n    cat << EOM\nerror: The sandbox is not in sync with the Podfile.lock. Run 'pod install' or update your CocoaPods installation.\nEOM\n    exit 1\nfi\n";
+			showEnvVarsInLog = 0;
+		};
```
Comment (maicki) [inline AsyncDisplayKit.xcodeproj/project.pbxproj]: Remove

**Comment by hannahmbanana** [comment]
@maicki: addressed your comments

File: examples/Videos/Sample/ViewController.m [hunk 5]
```diff
@@ -54,19 +54,19 @@ - (void)viewDidLoad
...
-    nicCageVideoNode.style.size = ASLayoutableSizeMakeFromCGSize(CGSizeMake(mainScreenBoundsSize.width/2, mainScreenBoundsSize.height / 3.0));
+    [nicCageVideoNode.style setSizeWithCGSize:CGSizeMake(mainScreenBoundsSize.width/2, mainScreenBoundsSize.height / 3.0)];
     nicCageVideoNode.style.layoutPosition = CGPointMake(mainScreenBoundsSize.width / 2.0, mainScreenBoundsSize.height / 3.0);
     
-    simonVideoNode.style.size = ASLayoutableSizeMakeFromCGSize(CGSizeMake(mainScreenBoundsSize.width/2, mainScreenBoundsSize.height / 3.0));
+    [simonVideoNode.style setSizeWithCGSize:CGSizeMake(mainScreenBoundsSize.width/2, mainScreenBoundsSize.height / 3.0)];
     simonVideoNode.style.layoutPosition = CGPointMake(0.0, mainScreenBoundsSize.height - (mainScreenBoundsSize.height / 3.0));
     
-    hlsVideoNode.style.size = ASLayoutableSizeMakeFromCGSize(CGSizeMake(mainScreenBoundsSize.width / 2.0, mainScreenBoundsSize.height / 3.0));
+    [hlsVideoNode.style setSizeWithCGSize:CGSizeMake(mainScreenBoundsSize.width / 2.0, mainScreenBoundsSize.height / 3.0)];
```
Comment (appleguy) [inline examples/Videos/Sample/ViewController.m]: This is a stylistic change that shouldn't actually be required, but I would tend to agree is a better style / better example, so it's fine to have here

**Comment by maicki** [comment]
LGTM

**Pull request merged**